tsl/test/sql/dist_util.sql

```diff
@@ -278,6 +278,19 @@ SELECT * FROM chunk_compression_stats('nondisttable') ORDER BY chunk_schema, chu
 SELECT * FROM hypertable_index_size('disttable_pkey');
 SELECT * FROM hypertable_index_size('nondisttable_pkey');
 
+-- Make sure timescaledb.ssl_dir and passfile gucs can be read by a non-superuser
+\c :TEST_DBNAME :ROLE_1
+\unset ECHO
+\o /dev/null
+SHOW timescaledb.ssl_dir;
```

Comment:
The output isn't shown in the test. Is this intentional. Now it is hard to see that this works.

Reply:
This is intentional. Otherwise test would be non-deterministic, because it shows full path which depends on an environment. In this test case we are interested only that the guc is accessible, without this fix it would generate and error and stop execution.

Reply:
I think the default output for `timescaledb.ssl_dir` is ""? Which should be ok and will be deterministic. But I do get your point that if permissions are not there then it would error out.